README.md

```diff
@@ -430,6 +431,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
 The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
 option.
 
+## Build Instructions
+
+Building your own images allows you to edit the configuration files of each image individually, providing more customization for your deployment.
+
+The docker images can be built by running the `make` command in the main repository folder. If you need to overwrite existing images from the remote source, use `FORCE_REBUILD=1 make`.
+
+If you are on the unstable branch, build the images with `FORCE_REBUILD=1 JITSI_RELEASE=unstable make`.
+
+Before deployment, make sure you remove the config file. You are now able to run `docker-compose up` as usual.
```

Comment:
What config file? You should need to remove anything.

Reply:
Ah, that was in regard to any changes done in `.env`, which requires the removal of `.jitsi-meet-cfg`. But that doesn't matter if we are only building the docker images here, so out of scope for the build process itself. Agree?

Reply:
Agreed!

Disregard the spacing thing, I see it ok now